src/lib/utils/skin.ts

```diff
@@ -108,3 +108,9 @@ export function isSkin(asset: Asset): boolean {
         (a) => a.category === 'Weapon' || (a.category === 'Type' && a.internal_name === 'Type_Hands')
     );
 }
+
+export function isMarketSkin(asset: Asset) {
```

Comment:
I think we can merge this with `isSkin`. Make the implementation of `isSkin` a OR between the tag parsing and this.

Reply:
i don't like that because i lowkey don't want to subject the inventory to this hacky check. perhaps i only fall to this if `asset.tags` is undefined?

Reply:
so it'd look like
```
if (asset.tags) {
   // inventory check
} else {
  // market check
}
```

Reply:
honestly it does the same thing but it short circuits if tags are defined

Reply:
Yeah, that sounds good to me.